config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
+  get 'users' => 'users#index'
```

Comment:
aqui pode ser usado `resources` https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default